data/assets/scene/solarsystem/heliosphere/bastille_day/density_volume.asset

```diff
@@ -0,0 +1,59 @@
+local assetHelper = asset.require("util/asset_helper")
+local sunTransforms = asset.require('scene/solarsystem/sun/transforms')
+
+local densityDirectory = asset.syncedResource({
+    Name = "Bastille Day MAS Density",
+    Type = "HttpSynchronization",
+    Identifier = "bastille_day_mas_density",
+    Version = 1
+})
+
+local sunRadius = 6.957E8
+
+local densityVolume = {
+    Identifier = "MAS_MHD_density", 
+    Parent = sunTransforms.SunIAU.Identifier,
+    Renderable = {
+        Type = "RenderableTimeVaryingVolume",
+        StepSize = 0.004,
+        --Enabled = false,
+        Opacity = 0.3,
+        TransferFunction = asset.localResource("transferfunctions/mas_mhd_r_squared.txt"),
+        SourceDirectory = densityDirectory,
+        --LowerValueBound = 0,
+        --UpperValueBound = 1,
+        GridType = "Spherical",
+        SecondsBefore = 24*60*60,
+        SecondsAfter = 24*60*60,
+        Dimensions = {100, 100, 128},
+        InvertDataAtZ = true
+    },
+    Transform = {
+        Scale = {
+            Type = "StaticScale",
+            Scale = sunRadius
+        },
+        Rotation = {
+            --Type = "StaticRotation", --LuaRotation
+            Type = "FixedRotation",
+                XAxis = {-1.0, 0.0, 0.0},
+                YAxis = {0.0, 1.0, 0.0},
+                ZAxis = {0.0, 0.0, -1.0}
+        }       
+    },
+    GUI = {
+        Name = "MAS Density",
+        Path = "/Solar System/Heliosphere"
+    }
+}
+
+assetHelper.registerSceneGraphNodesAndExport(asset, { densityVolume })
+
+asset.meta = {
+    Name = "Predictive Science Inc. Volume Rendering Bastille Day",
+    Version = "1.0",
+    Description = " Volumetric rendering for the bastille day CME event ",
+    Author = "CCMC, OpenSpace team",
+    URL = "https://dx.doi.org/10.3847/1538-4357/aab36d",
+    License = ""
+}
```

Comment:
Missing empty line a tend of file (Github marks them out clearly, so add a line everywhere this symbol is showed)